<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureWakaTimeDialog</class>
 <widget class="QDialog" name="ConfigureWakaTimeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>169</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>130</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configure WakaTime</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>361</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="KLineEdit" name="lineEdit_apiKey">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>301</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>36</number>
   </property>
   <property name="showClearButton" stdset="0">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="KLineEdit" name="lineEdit_apiUrl">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>301</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>36</number>
   </property>
   <property name="showClearButton" stdset="0">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>59</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>API key:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_hideFilenames">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>361</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Hide filenames</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>59</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>API url:</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigureWakaTimeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigureWakaTimeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
